src/cli.ts

```diff
@@ -189,9 +191,9 @@ if (options.token) {
 exitWithErrorIf(!options.folder, "--folder must be specified");
 exitWithErrorIf(!options.fromImage, "--fromImage must be specified");
 exitWithErrorIf(!options.toImage, "--toImage must be specified");
-exitWithErrorIf(!options.toRegistry && !options.toTar, "Must specify either --toTar or --toRegistry");
+exitWithErrorIf(!options.toRegistry && !options.toTar && !options.toDocker, "Must specify either --toTar, --toRegistry or --toDocker");
 exitWithErrorIf(
-	!options.toRegistry && !options.toToken && !options.toTar,
+	!options.toRegistry && !options.toToken && !options.toTar &&!options.toDocker,
```

Comment:
issue: When building to a tarball and an external registry the token-check is "ignored" here. 
suggestion: I think `!!options.toRegistry && !options.toToken` would solve this.